apps/vscode-extension/src/intellisense/design-token.ts

```diff
@@ -18,20 +18,25 @@ type DesignTokenCache = {
   tokenStructure: DesignTokenVariableSet;
 };
 
-const readFile = async (filePat: string) => (await vscode.workspace.fs.readFile(vscode.Uri.parse(filePat))).toString();
```

Comment:
is this forced by a linter rule ?

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v55.0.0/docs/rules/no-await-expression-member.md

Reply:
fallback change for this rule 
as we will discuss it later